srcpkgs/pinebookpro-base/files/60-pinebookpro.rules

```diff
@@ -1,7 +1,7 @@
 ACTION=="remove", GOTO="pinebookpro_end"
 KERNEL!="event*", GOTO="pinebookpro_end"
 
-ENV{idVendor}=="258a", ENV{idProduct}=="001e", ENV{ID_INPUT_KEYBOARD}=="1", \
+ATTR{idVendor}=="258a", ATTR{idProduct}=="001e", ENV{ID_INPUT_KEYBOARD}=="1", \
```

Comment:
I see `ATTRS` in most of my local rules...

Reply:
crap thanks for checking that